tests/genquad_test.py

```diff
@@ -0,0 +1,26 @@
+"""Tests for the general quadratic algebra drudge"""
```

Comment:
Nit, the title line is generally ended with a period as a sentence.

Reply:
```suggestion
"""Tests for the general quadratic algebra drudge."""
```